src/utils/text.h

```diff
@@ -120,11 +120,24 @@ namespace mbs
 
     MbsIterator & operator++()
     {
+      auto setToEnd = [&](){
```

Comment:
Why a lamda and not a private inlined method?

Reply:
Habits, inlined code can cause trouble especially for libraries. E.g. if you have code in header the symbols are not exported. So when you change it and applications using your library are not recompiled when a new version was released you end up with 2 versions of the function. One in the binary and one in the library.

Reply:
Which of course it irrelevant in code that is already in a header ;)